randovania/games/am2r/json_data/header.json

```diff
@@ -4296,6 +4296,92 @@
             "to_shuffle_proportion": 1.0
         }
     },
+    "used_trick_levels": {
```

Comment:
Does this mean, that as the DB is maturing, that i need to manually add the trick levels here?

Reply:
No, this is updated automatically when you save.

Reply:
I thought of making this only show up in the binary version of the db. Could still do that.

Reply:
Ah okay, must've missed it.